### DISCUSSION TITLE
Not allowed to load local resource

### DISCUSSION THREAD

**Discussion by gorilacrocodille** [perspective-dev/perspective#2640] [Q&A]
When I try to run this, I get an error and cannot understand where I went wrong. I would appreciate any help.

Env: Vite, Vue, v2.10.1

**Error:**
<img width="870" alt="error" src="https://github.com/finos/perspective/assets/124624310/0403c2b5-0d56-475d-a37b-430635750062">

**Comment by timkpaine** [comment]
Those errors are complaining about `.map` files. Please provide a repro, ideally a public repo that reproduces the issue. A screenshot is not debuggable.

**Reply by gorilacrocodille** [reply]
https://github.com/gorilacrocodille/perspective-vue/

**Reply by texodus** [reply]
Seems to work perfectly fine:

<img width="2056" alt="Screenshot 2024-06-15 at 11 44 00 AM" src="https://github.com/finos/perspective/assets/60666/1ea8a8b1-16ec-4440-8fb2-5d61b3e929c2">

**Reply by gorilacrocodille** [reply]
Yes, you are right, I found out that this error only occurs in Safari. In Chromium, there are no such errors
<img width="1680" alt="safari" src="https://github.com/finos/perspective/assets/124624310/0ac4fc3b-4e42-4b63-bae5-42b93fa6eb39">
